src/web/releases.rs

```diff
@@ -1001,4 +1003,59 @@ mod tests {
             Ok(())
         })
     }
+
+    #[test]
+    fn serialize_releases() {
+        let time = time::get_time();
+        let mut release = Release {
+            name: "serde".to_string(),
+            version: "0.0.0".to_string(),
+            description: Some("serde makes things other things".to_string()),
+            target_name: Some("x86_64-pc-windows-msvc".to_string()),
+            rustdoc_status: true,
+            release_time: time,
+            stars: 100,
+        };
+
+        let correct_json = json!({
+            "name": "serde",
+            "version": "0.0.0",
+            "description": "serde makes things other things",
+            "target_name": "x86_64-pc-windows-msvc",
+            "rustdoc_status": true,
+            "release_time": duration_to_str(time),
+            "release_time_rfc3339": time::at(time).rfc3339().to_string(),
```

Comment:
`time::at(time)` is really confusing, can you use `now` for the local variable instead?

Reply:
Can do, it's not ideal because `time` itself isn't ideal, but yet another PR